app/models/manageiq/providers/base_manager.rb

```diff
@@ -2,6 +2,8 @@ module ManageIQ::Providers
   class BaseManager < ExtManagementSystem
```

Comment:
This class is a bit of a wart we should clean up.. but for now, the below changes should probably go on `ExtManagementSystem` instead.